tests/Conformance.Tests/DataReaderTests.cs

```diff
@@ -12,8 +12,5 @@ public DataReaderTests(SelectValueFixture fixture)
 
 		[Fact(Skip = "Deliberately throws InvalidCastException")]
 		public override void GetTextReader_returns_empty_for_null_String() { }
-
-		[Fact(Skip = "https://github.com/mysql-net/MySqlConnector/issues/678")]
-		public override void GetSchemaTable_throws_after_Delete() { }
```

Comment:
I added this test to the PR, which tests the completed functionality. (It's currently failing.)